src/test/groovy/io/elastic/jdbc/actions/SelectMySQLSpec.groovy

```diff
@@ -27,6 +25,8 @@ class SelectMySQLSpec extends Specification {
   def databaseName = System.getenv("CONN_DBNAME_MYSQL")
   @Shared
   def host = System.getenv("CONN_HOST_MYSQL")
+  @Shared
+  def port = "3306"
```

Comment:
It makes sense to replace hardcoded port's value with ENV `System.getenv("CONN_PORT_MYSQL")`